### ISSUE TITLE
Blacklist not working?

### ISSUE THREAD

**Issue by Anahkiasen** [zalmoxisus/redux-devtools-extension#80]
Maybe I'm doing it wrong, I instrument the extension like this:

```js
const enhancers = compose(
        middlewares,
        reset(),
        window.devToolsExtension ? window.devToolsExtension({
            actionsBlacklist: ['REDUX_STORAGE_SAVE', 'game/NEW_CYCLE'],
        }) : f => f,
    );
```

But I still see those actions in the devtools:

![screen shot 2016-03-27 at 22 09 20](https://cloud.githubusercontent.com/assets/1321596/14067620/95e80150-f468-11e5-8d45-1fa2d886b39d.png)

Tried blacklisting those actions in the extension's options but same thing. Does the devtools call need to be at a certain position in the enhancers or something?

**Comment by zalmoxisus** [comment]
Thanks for the report!

In the first case you're doing everything right, this feature was just [added 2 days ago](https://github.com/zalmoxisus/redux-devtools-extension/pull/79), it is not on Chrome store yet. 

When you was trying to blacklist the actions in the extension's options, you probably forget to apply the checkbox as well.

Just published `1.0.0.17`, it should be available in few hours.

**Comment by Anahkiasen** [comment]
Works fine now, thanks!
